lib/encoder_impl.cc

```diff
@@ -443,7 +464,7 @@ void encoder_impl::prepare_group4a(void) {
 		((toffset>0?0:1)<<5)|((abs(toffset*2))&0x1f);
 }
 
-// for now single-group only
+// TMC Alert-C - http://tinyurl.com/alert-c-coding-handbook
```

Comment:
oh you're right. It should be http://forum.pocketnavigation.de/index.php?page=Attachment&attachmentID=1074401&h=538ea165b7f14bcb4165f989ec406d15eb979118
(although i doubt that that the tinyurl is less stable than the original document. but I get your point of course)